python/lsst/display/matplotlib/matplotlib.py

```diff
@@ -679,13 +816,30 @@ def __init__(self, minimum=0, dataRange=1, Q=8):
 
         See Lupton et al., PASP 116, 133
         """
-        Normalize.__init__(self)
-
         # The object used to perform the desired mapping
         self.mapping = afwRgb.AsinhMapping(minimum, dataRange, Q)
 
+        vmin, vmax = self._getMinMaxQ()[0:2]
+        if vmax*Q > vmin:
+            vmax *= Q
+        Normalize.__init__(self, vmin, vmax)
```

Comment:
Use `super().__init__(vmin, vmax)`

Reply:
Python2 code.  I'll make all the super changes requested